test/ctia/entity/web_test.clj

```diff
@@ -198,8 +198,8 @@
                        :confidence "Low"
                        :source "test"
                        :tlp "green"
-                       :owner "Unknown"
-                       :groups ["Administrators"]
+                       :owner "foouser"
+                       :groups ["foogroup"]
```

Comment:
I have no idea what triggered this test difference. Does anyone know if this change makes sense? Is it a regression?

Looking at the surroundings, it seems like "foouser" and "foogroup" are better expected values here, but I'm not familiar with the code.

```clojure
     (helpers/set-capabilities! "foouser" ["foogroup"] "user" all-capabilities)
     (whoami-helpers/set-whoami-response "45c1f5e3f05d0" "foouser" "foogroup" "user")
```